crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -835,6 +835,8 @@ impl CheckpointBuilder {
                 previous_gas_costs.computation_cost + current_gas_costs.computation_cost,
                 previous_gas_costs.storage_cost + current_gas_costs.storage_cost,
                 previous_gas_costs.storage_rebate + current_gas_costs.storage_rebate,
+                previous_gas_costs.non_refundable_storage_fee
```

Comment:
It looks like we could just make an `Add` implementation to the GasCostSummary and change this code to `previous_gas_costs + current_gas_costs`

Reply:
I'll consider thanks @andll 
for this PR (that is going crazy) maybe I'll stop making changes for now